frontend/src/component/admin/apiToken/ApiTokenForm/SelectProjectInput/SelectAllButton/SelectAllButton.tsx

```diff
@@ -1,28 +1,25 @@
 import React, { FC } from 'react';
-import { Box, Link } from '@mui/material';
-import { useStyles } from './SelectAllButton.styles';
+import { Box, Link, styled } from '@mui/material';
 
 type SelectAllButtonProps = {
     isAllSelected: boolean;
     onClick: () => void;
 };
 
+const StyledLink = styled(Link)(({ theme }) => ({
+    cursor: 'pointer',
+    fontSize: theme.fontSizes.bodySize,
+})) as typeof Link;
```

Comment:
I'd inject a function to sx on the element itself instead to avoid type casting. Please check how we used themeStyles/focusable in a few places.